machine/v1beta1/types_vsphereprovider.go

```diff
@@ -172,6 +179,15 @@ type NetworkDeviceSpec struct {
 	AddressesFromPools []AddressesFromPool `json:"addressesFromPools,omitempty"`
 }
 
+// VSphereDisk describes additional disks for vSphere.
+type VSphereDisk struct {
+	// The device name exposed. Used to identify the disk.
+	// +optional
+	DeviceName *string `json:"deviceName,omitempty"`
+	// SizeGB is the size of the disk (in GB).
```

Comment:
Serialised version to start.

What's the minimum and maximum values here?

Do we mean GB or GiB?

Reply:
Good catch.  I'll update that.